applications/smart-distancing/ui/utils/visualization_utils.py

```diff
@@ -390,6 +391,38 @@ def visualize_boxes_and_labels_on_image_array(
     return image
 
 
+
+def checkDistance(nn_out):
```

Comment:
Please don't use camelCase format for defining functions and methods
e.g. distance_checker, check_distance, ...